README.md

```diff
@@ -155,6 +169,12 @@ branches:
         apps: []
         users: []
         teams: []
+      # Permits force pushes for all users with push access. Set to null to disable.
```

Comment:
i dont think "disable" makes sense for this one

Reply:
would you like to change "disable" to "allow"?